tests/ui/test_ui_tools.py

```diff
@@ -1515,17 +1516,35 @@ def test_private_message_to_self(self, mocker):
         ('<blockquote>stuff', [('msg_quote', ['', 'stuff'])]),
         ('<div class="message_embed">',
             ['[EMBEDDED CONTENT NOT RENDERED]']),  # FIXME Unsupported
-        ('<a href="http://foo">http://foo</a>', [('msg_link', 'http://foo')]),
+        ('<a href="http://foo">Foo</a><a href="https://bar.org">Bar</a>',
+         [('msg_link', 'Foo'), ' ', '[1]',
+          ('msg_link', 'Bar'), ' ', '[2]']),
+        ('<a href="http://foo">Foo</a><a href="http://foo">Another foo</a>',
+         [('msg_link', 'Foo'), ' ', '[1]',
+          ('msg_link', 'Another foo'), ' ', '[1]']),
```

Comment:
minor: A variant of this showing the stripping of trailing slashes (maybe even with identical text?) would be useful, as I don't think we have a test of that?